test/models/rental_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+<<<<<<< HEAD
+#
+=======
+
+>>>>>>> d895f891acf50da64888798d075b4c23e843b99f
```

Comment:
Why are there git merge markers in this file?

Reply:
If missing these markers is a regular problem for you, you might want to set up a pre-commit hook:
https://jondowdle.com/2015/02/block-merge-conflicts-in-commits/